package.json

```diff
@@ -21,7 +21,7 @@
     "compile": "env NODE_ENV=production concurrently yarn:compile:*",
     "compile:main": "yarn run webpack --config webpack/main.ts",
     "compile:renderer": "yarn run webpack --config webpack/renderer.ts",
-    "compile:extension-types": "yarn run tsc --project tsconfig.extension-api.json",
```

Comment:
Where is `tsconfig.extension-api.json` used after this change?

Reply:
It is not, I guess I should remove it. I think in the future we might return to using tsc if we no longer output anything other than types from the extension API.